### PR TITLE
08:29 - Alter in remote - github-other

### PR DESCRIPTION
**Pull request by FlyDogBR**
08:29 - Alter in remote - github-other

Created a new pull request by other github user

### CODE REVIEW

**Comment by osvaldo-github** [comment]
Nice, that´s first review pull request to me.

File: document4.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-[DEVELOP] 2019-02-22 - 15:48 - Alter in local
\ No newline at end of file
+[DEVELOP] 2019-02-22 - 15:48 - Alter in local
+[DEVELOP] 2019-03-01 - 08:29 - Alter in remote - github-other
```
Comment (osvaldo-github) [inline document4.txt]: Correct this line

Comment (FlyDogBR) [reply document4.txt]: Below is correct?

```suggestion
[DEVELOP] 2019-03-01 - 08:29 - Alter in remote - github FlyDogBR
```

Comment (osvaldo-github) [reply document4.txt]: Yes, correct.

Comment (FlyDogBR) [reply document4.txt]: I changed the document

**Review by osvaldo-github — CHANGES_REQUESTED** [review]
Ok, revised.

**Review by osvaldo-github — APPROVED** [review]

**Pull request merged**